analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs

```diff
@@ -32,7 +32,7 @@ namespace SonarAnalyzer.Rules.VisualBasic
     [DiagnosticAnalyzer(LanguageNames.VisualBasic)]
     public sealed class VariableUnused : VariableUnusedBase
     {
-        private static readonly DiagnosticDescriptor rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+        private static readonly DiagnosticDescriptor rule = DescriptorFactory.Create(DiagnosticId, MessageFormat);
```

Comment:
You keep the lowecase variations lowercase?

Reply:
Long-term: No, we'll fix them.

For this PR (and the batch 2): I do not fix anything to keep the PR as small as possible. Also as regular as possible with no sideeffects.

Reply:
Logical choice. Just checking.